src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -2557,7 +2557,21 @@ public ServiceCommandException(string message, Exception innerException)
         {
         }
         #endregion ctors
-        
+
+        #region Serialization
+        /// <summary>
+        /// Serialization constructor.
+        /// </summary>
+        /// <param name="info"></param>
+        /// <param name="context"></param>
+        /// <returns>Constructed object.</returns>
+        protected ServiceCommandException(SerializationInfo info, StreamingContext context)
```

Comment:
should we mark this obsolete so there is a compile time error

And other similar cases?

Reply:
done